app/frontend/javascripts/classes/StoreManagerMixin.js

```diff
@@ -333,6 +334,18 @@ export const mixin = {
           }
         })
         .catch((e) => {
+          const body = document.querySelector('body');
+          const modalWindow = new ModalWindow(body, e.message);
+          modalWindow.show();
+
+          body.addEventListener('click', (e) => {
+            if (e.target !== modalWindow.messageArea) {
+              modalWindow.hide();
+              body.removeEventListener('click', clickHandler);
+            }
+          });
+          body.addEventListener('click', clickHandler);
```

Comment:
すみません、前回わかりずらかったかと思います、こうやって：
```suggestion
         const clickHandler = (e) => {
            if (e.target !== modalWindow.messageArea) {
              modalWindow.hide();
              body.removeEventListener('click', clickHandler);
            }
          }
          body.addEventListener('click', clickHandler);
```